### ISSUE TITLE
Sort on Pivot only

### ISSUE THREAD

**Issue by Gummby** [optimistdigital/nova-sortable#16]
Hi there, 

Not sure if this intended behaviour. But I want to sort only on relationships, so when it is on relationship/pivot table.

This below will return true even if it's not viaRelationship as the second property is set true in my model.

`$sortable  = $request->viaRelationship() ? $request->newResource()->resource->sortable['sort_on_pivot'] ?? false : true;`

am I using it wrong or is this just how its meant to work.

I can fix this easily by over riding the serializeForIndex method though so no real action is needed if this intended behaviour.

On laravel 7 and Nova 3

**Comment by Tarpsvo** [comment]
Hi! This is the intended behaviour as I haven't added a flag for the "show only on pivot tables" use-case.

If you have any ideas on how to handle this, input is greatly appreciated!

**Comment by Gummby** [comment]
Hi, I just override the serializeForIndex method and made it a little more verbose 

```
    public function serializeForIndex(NovaRequest $request, $fields = null)
    {

        if ($request->viaRelationship() && $request->newResource()->resource->sortable['sort_on_pivot']) {
            return array_merge(parent::serializeForIndex($request, $fields), [
                'sortable' => true,
            ]);
        } else {
            return array_merge(parent::serializeForIndex($request, $fields), [
                'sortable' => false,
            ]);
        }

    }
```

This means it wont show the sort controls on the resources index view on the relationship index view

**Comment by Tarpsvo** [comment]
Hi! The newly released version 1.4.0 supports this exact behaviour - you can only enable sorting on either the Index view OR the HasMany/BelongsToMany/MorphsMany relationship, but not both. Let me know if it works as intended for you now. Good luck!
